{
  "cve_id": "CVE-2023-39661",
  "timestamp": "2025-01-14 16:58:43",
  "phases": {
    "primary": {
      "completed": true,
      "timestamp": "2025-01-10 10:39:54"
    },
    "secondary": {
      "completed": true,
      "timestamp": "2025-01-11 20:55:42"
    },
    "extraction": {
      "completed": true,
      "timestamp": "2025-01-14 14:08:20"
    }
  },
  "urls": [
    {
      "url": "https://github.com/gventuri/pandas-ai/issues/410",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    }
  ]
}